features/logging/kibana.feature

```diff
@@ -92,6 +92,13 @@ Feature: Kibana related features
     When I run the :check_log_count web action
     Then the step should succeed
     """
+    # Verify the token are encrypted in kibana logs 
+    Given the first user is cluster-admin
+    Given a pod becomes ready with labels:
+            | logging-infra=kibana |
+    When I run the :logs admin command with:
+      | resource_name | <%= pod.name %>                |
+    Then the output should not contain "XXXXXXXXXXXXXXXXXXXXXXXX"
```

Comment:
Hi @anpingli , what is "XXXXXXXXXXXXXXXXXXXXXXXX"?

Reply:
@QiaolingTang  That is the encrypted token.

Reply:
{"type":"response","@timestamp":"2021-03-29T09:38:29Z","tags":[],"pid":118,"method":"post","statusCode":200,"req":{"url":"/elasticsearch/_msearch?rest_total_hits_as_int=true&ignore_throttled=true","method":"post","headers":{"host":"kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com","user-agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.96 Safari/537.36","content-length":"955","accept":"application/json, text/plain, */*","accept-encoding":"gzip, deflate, br","accept-language":"en-US,en;q=0.9","content-type":"application/x-ndjson","forwarded":"for=66.187.232.127;host=kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com;proto=https","kbn-version":"6.8.1","origin":"https://kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com","referer":"https://kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com/app/kibana","sec-fetch-dest":"empty","sec-fetch-mode":"cors","sec-fetch-site":"same-origin","x-forwarded-access-token":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX","x-forwarded-email":"XXXXXXXXXXXXXXXXXXXXXXXX","x-forwarded-for":"66.187.232.127, 10.129.2.21","x-forwarded-host":"kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com","x-forwarded-port":"443","x-forwarded-proto":"https","x-forwarded-user":"testuser-0","securitytenant":"__user__"},"remoteAddress":"127.0.0.1","userAgent":"127.0.0.1","referer":"https://kibana-openshift-logging.apps.anli47.qe.gcp.devcluster.openshift.com/app/kibana"},"res":{"statusCode":200,"responseTime":75,"contentLength":9},"message":"POST /elasticsearch/_msearch?rest_total_hits_as_int=true&ignore_throttled=true 200 75ms - 9.0B"}